public/pages/forgot-password.html

```diff
@@ -0,0 +1,103 @@
+                        <tr>
+                          <td align="center" style="word-break: break-word; font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif; box-sizing: border-box;">
+                            <!-- Border based button
+                       https://litmus.com/blog/a-guide-to-bulletproof-buttons-in-email-design -->
+                            <table width="100%" border="0" cellspacing="0" cellpadding="0" style="font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif; box-sizing: border-box;">
+                              <tr>
+                                <td align="center" style="word-break: break-word; font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif; box-sizing: border-box;">
+                                  <table border="0" cellspacing="0" cellpadding="0" style="font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif; box-sizing: border-box;">
+                                    <tr>
+                                      <td style="word-break: break-word; font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif; box-sizing: border-box;">
+                                        <a href="http://{{host}}/auth/reset/{{token}}" class=" button  button--green " target="_blank" style="font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif; box-sizing: border-box; display: inline-block; color: #FFF; border-radius: 3px; box-shadow: 0 2px 3px rgba(0, 0, 0, 0.16); -webkit-text-size-adjust: none; background-color: #22BC66; border-top: 10px solid #22BC66; border-right: 18px solid #22BC66; border-bottom: 10px solid #22BC66; border-left: 18px solid #22BC66; text-decoration: none;">Reset your password</a>
+                                      </td>
+                                    </tr>
+                                  </table>
+                                </td>
+                              </tr>
+                            </table>
+                          </td>
+                        </tr>
+                      </table>
+                      <p style="margin-top: 0; color: #74787E; font-size: 16px; line-height: 1.5em; text-align: left; font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif; box-sizing: border-box;">If you did not request a password reset, please ignore this email or <a href="{{support_url}}" style="color: #3869D4; text-decoration: none; font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif; box-sizing: border-box;">contact support</a> if you have questions.</p>
+                      <p style="margin-top: 0; color: #74787E; font-size: 16px; line-height: 1.5em; text-align: left; font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif; box-sizing: border-box;">Thanks,
+                        <br>The Peace Corps Team</p>
```

Comment:
Oh okay @visitha. I have done it for angular files. Will do it for the node files as well.